### ISSUE TITLE
New Feature: Femur gap distance

### ISSUE THREAD

**Issue by ninanikitina** [cs481-ekh/f22-ai-cbl#152]

**Comment by ninanikitina** [comment]
Add femur gap distance feature. Shown in red

![Image](https://user-images.githubusercontent.com/68862634/202269128-21f56b07-7d2a-4f20-ba21-5c275d6ec434.png)

**Comment by ninanikitina** [comment]
Timing:
~ 9 hours

The general idea is shown below. I used two slices of red and green. Initially, I tried to find the red and greed outline interesting using pyvista instruments, but it did not work. So I found minimal distances between red and green point clouds in a specified region. 
omplex_features_exploration.ipynb shows distances for set 2. It can be seen that some femur distance is not ideal. I tried to play with the parameters and spent two hours, but some corrections are still needed. 


![Image](https://user-images.githubusercontent.com/68862634/204365050-6532f812-5cb0-449c-b913-b9fd31f47747.png)



![Image](https://user-images.githubusercontent.com/68862634/204365060-eea20f31-7968-4a23-8a78-20a96f66e5f9.png)
